config.yaml

```diff
@@ -4,3 +4,4 @@ ratsd:
   listen-addr: 0.0.0.0:8895
   protocol: http
   plugin-dir: attesters/bin
+  list-options: all
```

Comment:
I am probably missing something, but I don’t fully grasp why this should be configuration-defined behaviour, rather than being completely driven by the API user.

Reply:
I was trying to implement attester discovery with such config option. [In your proposal](https://github.com/veraison/ratsd/wiki/API-custom-parameters#client-api), you mentioned the user have to specify at least `"attester-id": null` to include the evidence from `attester-id`. The option was added to make ratsd behave the same way as the current implementation without this PR. But I'd agree the user may want to perform the discovery first before the query and without restarting ratsd.

Reply:
> I was trying to implement attester discovery with such config option. [In your proposal](https://github.com/veraison/ratsd/wiki/API-custom-parameters#client-api), you mentioned the user have to specify at least `"attester-id": null` to include the evidence from `attester-id`.

Oh, thanks. I forgot I had written that thing 😓 

> The option was added to make ratsd behave the same way as the current implementation without this PR. But I'd agree the user may want to perform the discovery first before the query and without restarting ratsd.

Perhaps we could leave it in as a temp backstop and remove it when we have implemented proper discovery.

Reply:
@thomas-fossati I've updated the README to include attester selection. The decision to remove such option and refine the default behavior should be determined in the Attester Discovery PR IMO.